### PR TITLE
spatial: add page

### PR DESCRIPTION
**Pull request by andreadellacorte**
- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/spatial.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# spatial
+
+> A set of commands for managing and developing SpatialOS projects.
+
+- Build workers for local deployment on Unity on Windows:
```
Comment (sbrl) [inline pages/common/spatial.md]: "Build workers"? I'm not sure I understand here. Does this compile the source code for some kind of worker process?

Also, what does Unity have to do with an operating system?

File: pages/common/spatial.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+`spatial worker build --target=development --target=Osx`
+
+- Build workers for local deployment on Unreal:
+
+`spatial worker build --target=local --target=Windows`
+
+- Deploy locally:
+
+`spatial local launch {{launch_config}} --snapshot={{snapshot_file}}`
```
Comment (sbrl) [inline pages/common/spatial.md]: So these commands operate on the project in the current directory like [npm](https://npmjs.org/)?

File: pages/common/spatial.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+- Launch a local worker to connect to your local deployment:
+
+`spatial local worker launch UnityClient default`
+
+- Upload an assembly to use for cloud deployments:
+
+`spatial cloud upload {{assembly_name}}>`
```
Comment (sbrl) [inline pages/common/spatial.md]: Is the `>` here a typo?

File: pages/common/spatial.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# spatial
+
+> A set of commands for managing and developing SpatialOS projects.
+
+- Build workers for local deployment on Unity on Windows:
+
+`spatial worker build --target=development --target=Osx`
```
Comment (agnivade) [inline pages/common/spatial.md]: The target is osx here, but in the description you mention Windows. How does it work ?

File: pages/common/spatial.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+- Build workers for local deployment on Unreal:
+
+`spatial worker build --target=local --target=Windows`
```
Comment (agnivade) [inline pages/common/spatial.md]: Here your target is local, but in the previous example, it was development and still the description says "local deployment". Are the two not related ?

File: pages/common/spatial.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+- Deploy locally:
+
+`spatial local launch {{launch_config}} --snapshot={{snapshot_file}}`
+
+- Launch a local worker to connect to your local deployment:
+
+`spatial local worker launch UnityClient default`
```
Comment (agnivade) [inline pages/common/spatial.md]: Should UnityClient be in tokens `{{}}` ?

File: pages/common/spatial.md (hunk 2 above)
Comment (andreadellacorte) [reply pages/common/spatial.md]: Yes, that's correct

File: pages/common/spatial.md (hunk 1 above)
Comment (andreadellacorte) [reply pages/common/spatial.md]: They build Unity (a game engine) processes that can run on a Windows' platform

File: pages/common/spatial.md (hunk 5 above)
Comment (andreadellacorte) [reply pages/common/spatial.md]: Unreal and Unity use different targets

File: pages/common/spatial.md (hunk 6 above)
Comment (andreadellacorte) [reply pages/common/spatial.md]: UnityClient (as it is right now) is the default, but I can make it more generic and have it in {{}}

File: pages/common/spatial.md (hunk 3 above)
Comment (andreadellacorte) [reply pages/common/spatial.md]: Yes, thank you

File: pages/common/spatial.md (hunk 4 above)
Comment (andreadellacorte) [reply pages/common/spatial.md]: Mistake, will fix

File: pages/common/spatial.md (hunk 5 above)
Comment (agnivade) [reply pages/common/spatial.md]: My question was how does specifying the target change the deployment type. Here target=local, and you say "local deployment", which is fine. But above this, you have "target=development" but still it was "local deployment". 

So what does development and local targets mean when both of them are used for local deployments ?

File: pages/common/spatial.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
+# spatial
+
+> A set of commands for managing and developing SpatialOS projects.
+
+- Run this when you use a project for the first time:
+
+`spatial worker build`
+
+- Build workers for local deployment on Unity on on Osx:
```
Comment (agnivade) [inline pages/common/spatial.md]: "on" is repeated. And also, please change Osx to macOS.

**Comment by agnivade** [comment]
Just for visibility's sake (Github's review UI is bad), 2 of my comments are still unresolved.
- https://github.com/tldr-pages/tldr/pull/1652#discussion_r153069004
- https://github.com/tldr-pages/tldr/pull/1652#discussion_r153068973

File: pages/common/spatial.md (hunk 5 above)
Comment (andreadellacorte) [reply pages/common/spatial.md]: Hey! Both targets build for local development, but when building for Unreal you use `--target=local` and when building for Unity you use `--target=development`

I flagged this to the devs

File: pages/common/spatial.md (hunk 7 above)
Comment (andreadellacorte) [reply pages/common/spatial.md]: I think my latest check in fixes this [link](https://github.com/tldr-pages/tldr/pull/1652/commits/e953129676feb85c66907c12f31ecc3447328cb6)

File: pages/common/spatial.md (hunk 5 above)
Comment (agnivade) [reply pages/common/spatial.md]: I see. That is very confusing indeed.

**Comment by sbrl** [comment]
Thanks for the page, @andreadellacorte!

**Pull request merged**